src/test/java/com/bitrix/tests/BitRixTaskPageTestOC.java

```diff
@@ -0,0 +1,26 @@
+package com.bitrix.tests;
+
+import com.bitrix.base.TestBase;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class BitRixTaskPageTestOC extends TestBase {
+    @Test
+    public void test() {
+        loginPage.login("hr_user");
+
+        WebElement task = driver.findElement(By.xpath("//span[.='Task']"));
```

Comment:
I created PortalPage for this part - will/can delete later or update project - did you pull before pushing?